src/autoupdater.ts

```diff
@@ -478,6 +491,20 @@ export class AutoUpdater {
             return false;
           }
 
+          if (
+            'status' in e &&
+            (e as octokit.RequestError).status === 404) {
+            const error = e as Error;
+
+            ghCore.error(
+              `Unable to merge pull request #${prNumber}. Could be because the PR doesn't exist, or the token lacks write permissions for the repo. Error was: ${error.message}`,
```

Comment:
We encountered this "Not found" issue when configuring the action and using a token that only had read access for the repo. It doesn't need to be part of this PR of course if you prefer keeping it separate or not including it